### DISCUSSION TITLE
[BUG] Not Synced. Syncing always 20 min behind blockchain [v 1.2.10]

### DISCUSSION THREAD

**Discussion by genGrievouz** [Chia-Network/chia-blockchain#9144] [Support]
Using ubuntu server on raspberry pi 4B. Running newest version of OS. Everything is up to date.

After updating to 1.2.10 I can not catch up with blockchain. I have been updating client many times I have never had such problems. My connection is strong, ports are doing okey. I had added many IPs. I have restarted my harvester/farmer/node many time today. Nothing helps. 

Request an assistance.

**Comment by moonlitbugs** [comment]
You are not alone.  Although my node is synced, a lot of the nodes to which my node communicates are not.  Here's a quick example:

```console
~$ /usr/local/chia/peer-heights.sh
31.19.209.4     ←    2.3↑    0.0↓  -650 Δ
77.3.44.180     →   11.3↑    0.5↓    -2 Δ
77.181.127.150  →    2.3↑    5.8↓    -5 Δ
77.242.90.117   ←    1.5↑    1.6↓  1076933
79.131.100.222  →    9.0↑    0.4↓    -5 Δ
82.21.126.129   →   13.6↑    0.5↓    -9 Δ
82.75.105.4     →    8.8↑    0.3↓    -5 Δ
92.28.173.165   ←    0.9↑    7.7↓         
92.105.52.6     ←    0.3↑   17.6↓         
98.195.72.211   →    5.8↑    0.6↓   -28 Δ
108.41.132.61   →    0.1↑    0.2↓   -31 Δ
108.51.245.169  →    4.4↑    3.9↓    -4 Δ
138.68.186.44   ←    1.7↑   10.6↓    -5 Δ
142.54.161.107  ←    0.3↑    0.3↓   -18 Δ
155.133.91.176  ←    1.1↑    0.5↓    -5 Δ
162.229.180.100 →    5.0↑    0.0↓ -1626 Δ
172.90.19.73    →    9.7↑    0.8↓   -21 Δ
174.0.78.103    →   16.2↑    0.5↓   -11 Δ
176.123.14.19   →    5.3↑    0.6↓    -9 Δ
180.92.27.115   →    4.3↑    2.5↓    -8 Δ
185.54.100.107  →   12.7↑    3.7↓    -2 Δ
185.251.161.20  ←    2.2↑    5.0↓    -5 Δ
193.32.127.216  →    5.3↑    2.3↓    -5 Δ
217.65.138.115  →    2.7↑    0.5↓   -11 Δ
~$
```
Of all the peering nodes, only 3 are at the proper height (1076933).  Many are 5 blocks behind; some are far worse.

I think it would be great of one of the Chia devs could explain how the network of full-nodes is mapped out, topologically.  Clearly, large swathes are syncing to below-height.

**Comment by genGrievouz** [comment]
Indeed, I have many peers nodes behind.

**Comment by loppefaaret** [comment]
https://github.com/Chia-Network/chia-blockchain/discussions/9049
are you still having the same issue ? it might be that from 1.2.10 we default back to running the blockchain database in full sync mode, to minimize the amount of corruption we see in the user base(we tried lesser in 1.2.8 and 1.2.9)

**Reply by moonlitbugs** [reply]
I'm not the OP, but in answer, everything on my node cleared up after the dust storm settled.

I see "full sync" mentioned twice in the code (```full_node.py``` and ```wallet_node.py```) but did not realize there was an incremental version.  There is no mention of "*incremental*" anywhere in the ```CHANGELOG.md```.  I would love it if somebody would write up – and include along with the README.md – a document that describes, "*How Chia works under-the-hood*."  For sure, there is a lot of confusion amongst the userbase as to how network syncing works, what to expect, what to do when syncing appears stuck, and so forth.

**Reply by genGrievouz** [reply]
Sorry for delayed response. It works now (1.2.10). Thank you.
